src/pages/Host.tsx

```diff
@@ -16,8 +16,8 @@ export const Host: React.FC = () => {
   const { data, loading } = useQuery<HostQuery, HostQueryVariables>(GET_HOST, {
     variables: { id },
   });
-  const host = get(data, "host");
-  const hostUrl = get(host, "hostUrl");
+  const host = data?.host;
+  const hostUrl = host?.hostUrl;
   const status = get(host, "status") as HostStatus;
```

Comment:
I think you can also remove this instance of lodash.get and remove the import uptop

Reply:
oh haha yes!! thanks!